src/settings/MappingProfiles/detailsSections/edit/InvoiceDetailSection/InvoiceInformation.js

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  TextField,
+  TextArea,
+  Checkbox,
+} from '@folio/stripes/components';
+
+import {
+  AcceptedValuesField,
+  DatePickerDecorator,
+} from '../../../../../components';
+
+import {
+  getFieldName,
+  getAcceptedValuesPath,
+} from '../../utils';
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import { okapiShape } from '../../../../../utils';
+
+export const InvoiceInformation = ({
+  setReferenceTables,
+  okapi,
+}) => {
+  const [isLockTotal, setIsLockTotal] = useState(false);
+
+  return (
+    <Accordion
+      id="invoice-information"
+      label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.invoiceInformation.section`} />}
+    >
+      <Row left="xs">
+        <Col
+          data-test-invoice-date
```

Comment:
Are you sure you need this data-attribute? If you want to test it with RTL it's better to replace it with `data-testid` attribute.

Reply:
Forgot to remove. Done